### PR TITLE
issue-9: escaped more NA values in .plotFit

### PR DESCRIPTION
**Pull request by daynefiler**
Similar to issue-2, the new nchar behavior with
R3.3 was causing errors. Additional changes were
made to prevent NA values being sent to nchar that
were missed when fixing issue-2.

### CODE REVIEW

**Comment by daynefiler** [comment]
Matt, once you check this out we can merge it into master.

File: R/plotFit.R [hunk 1]
```diff
@@ -260,7 +260,7 @@
     paste0("ASSAY:   ", aenm, "\n\n",
            "NAME:    ", chnm, "\n",
            "CHID:    ", chid, spaces(8 - nchar(chid)),
-           "CASRN: ", casn, "\n", # spaces(16-nchar(casn)),"AGBY: ",agby,"\n",
```
Comment (ericwatt) [inline R/plotFit.R]: Did you want to delete the line above this instead? You've removed CASRN and have two lines for CHID

**Comment by daynefiler** [comment]
@ericwatt I probably deleted the wrong line -- I will take a look at it. I intended to just delete the commented rest of that line (it was from a very _very_ old version).

**Comment by daynefiler** [comment]
@martino165 @ericwatt Does it look alright now?

**Comment by martino165** [comment]
This looks great to me. I was going to recommend functionalizing the fix and it looks like you did that.

**Comment by ericwatt** [comment]
Yeah, that addresses my comment.

**Pull request merged**